wgs_somatic-run-wrapper.py

```diff
@@ -131,16 +133,35 @@ def wrapper():
                     n = k
                     n_ID = [val for val in pair_dict_all_pairs.get(k) if val != 'normal'][0]
                     if n_ID == t_ID or t_ID == n.split("DNA")[1] or n_ID == t.split("DNA")[1]: # if we change to pair id instead of tumorNormalID this is needed
+                        runnormal = Rctx_run.run_name
+                        runtumor = Rctx_run.run_name
```

Comment:
Necessary to have both of these with the same value?

Reply:
they are needed as arguments to start the pipeline